consensus/src/pipeline/virtual_processor/processor.rs

```diff
@@ -407,21 +418,23 @@ impl VirtualStateProcessor {
                 // Calling the drops explicitly after the batch is written in order to avoid possible errors.
                 drop(virtual_write);
 
-                // Stops consensus from sending into and bloating an unread channel in cases where event processor is not required (such as in testing cases).
-                if self.consensus_sender.receiver_count() > 0 {
-                    // We use try_send on consensus sender since this is none-blocking.
-                    self.consensus_sender
-                        .try_send(ConsensusEvent::VirtualChangeSet(Arc::new(VirtualChangeSetEvent {
-                            accumulated_utxo_diff: Arc::new(accumulated_diff),
-                            parents: Arc::new(new_virtual_state.parents),
-                            selected_parent_blue_score: new_virtual_state.ghostdag_data.blue_score,
-                            daa_score: new_virtual_state.daa_score,
-                            mergeset_blues: new_virtual_state.ghostdag_data.mergeset_blues,
-                            mergeset_reds: new_virtual_state.ghostdag_data.mergeset_reds,
-                            accepted_tx_ids: Arc::new(new_virtual_state.accepted_tx_ids),
-                        })))
-                        .expect("expected send");
-                };
+                // Emit notifications
+                let accumulated_diff = Arc::new(accumulated_diff);
+                let virtual_parents = Arc::new(new_virtual_state.parents);
+                let _ = self
+                    .notification_root()
+                    .notify(Notification::UtxosChanged(UtxosChangedNotification::new(accumulated_diff, virtual_parents)));
```

Comment:
Since `notify` calls do not require the cloned Arc, can we just directly access the `notification_root` field?